pykeeb/keyboard_matrix.py

```diff
@@ -35,6 +36,48 @@ def __init__(self, r, c, row_spacing=1.5, column_spacing=1.5, plate_thickness=3,
 		self.ik = self.ignore_keys = [[False for a in range(c)] for b in range(r)] 
 		self.generate()
 
+
+	def project_rows(self, R):
```

Comment:
Looks good.  However, I may change the name to arc_rows() to avoid confusion with the existing projection function.

Reply:
Great. I can write up a doc string.